src/csharp/Microsoft.Spark/Sql/DataFrame.cs

```diff
@@ -714,7 +714,11 @@ public IEnumerable<Row> Collect()
         /// Returns an iterator that contains all of the rows in this `DataFrame`.
         /// The iterator will consume as much memory as the largest partition in this `DataFrame`.
         /// </summary>
+        /// <remarks>
+        /// This API is removed in Spark 3.0.
+        /// </remarks>
         /// <returns>Row objects</returns>
+        [Removed(Versions.V3_0_0)]
```

Comment:
This is not removed. They changed the contract. We should follow this: https://github.com/apache/spark/blob/0a7095156bdb565133f7dcc74546c51a5e5d2414/python/pyspark/rdd.py#L175